app/src/main/res/values-es/strings.xml

```diff
@@ -329,13 +329,13 @@
     <string name="token_msg_uniToken_msg1">Esta ficha da una representación del IV de tu monstruo, pero utilizando carácteres UNICODE especiales que permiten mostrar números como 11 en un solo carácter, como ⑪. Esto ahorra espacio.</string>
     <string name="token_msg_uniToken_msg2">" Esta ficha usa carácteres rellenos, que tiene un interior negro en los números, como ⓭."</string>
     <string name="token_msg_uniToken_msg3">" Esta ficha usa carácteres vacíos, que tiene un interior blanco en los números, como ⑪."</string>
-    <string name="token_msg_worthTra_msg1">"Esta ficha devuelve la evaluación sobre cómo vale la pena entrenar este monstruo, basado tanto en las estadísticas base como en la "</string>
-    <string name="token_msg_worthTra_msg2" formatted="false">" estadística posible de IV. Por ejemplo, un Alakazam de 96% IV obtendrá una puntuación mayor que un Tyranitar de 20%, independientemente del hecho de que el PC máximo posible para este último sea mayor: ya que la probabilidad de mejorar Tyranitar es mayor que la de conseguir un Alakazam mejor, el segundo representa una inversión mucho mejor de polvoestelares. Rangos entre 00–99. Siempre devuelve dos dígitos."</string>
-    <string name="token_msg_train">Entrenar</string>
+    <string name="token_msg_worthTra_msg">"Esta ficha devuelve la evaluación sobre cómo vale la pena entrenar este monstruo, basado tanto en las estadísticas base como en la %1$s estadística posible de IV. Por ejemplo, un Alakazam de 96% IV obtendrá una puntuación mayor que un Tyranitar de 20%, independientemente del hecho de que el PC máximo posible para este último sea mayor: ya que la probabilidad de mejorar Tyranitar es mayor que la de conseguir un Alakazam mejor, el segundo representa una inversión mucho mejor de polvoestelares. Rangos entre 00–99. Siempre devuelve dos dígitos."</string>
```

Comment:
Please review this change, this causes an error since % symbols are not escaped (you should write a double percent %% to escape it).
If you run the app and check the worth training token description you'll understand what's not working :)

Reply:
Something like:
un Alakazam de 96%%
que un Tyranitar de 20%%